app/code/Magento/InventoryDownloadable/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "magento/module-inventory-downloadable",
+    "description": "N/A",
+    "require": {
+        "php": "7.0.2|7.0.4|~7.0.6|~7.1.0",
+        "magento/framework": "100.3.*",
+        "magento/module-downloadable": "100.3.*"
+    },
+    "suggest": {
+        "magento/module-inventory-configuration": "100.0.*"
```

Comment:
Looks like after last discussion
We need to remove this dependency

Reply:
Maybe i should remove module InventoryDownloadable?